llvm/test/CodeGen/X86/apx/cet.ll

```diff
@@ -0,0 +1,50 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc < %s -mtriple=x86_64-apple-darwin -mattr=+shstk,+egpr --show-mc-encoding | FileCheck %s
+
+define void @test_wrssd(i32 %a, ptr %__p) {
+; CHECK-LABEL: test_wrssd:
+; CHECK:       ## %bb.0: ## %entry
+; CHECK-NEXT:    wrssd %edi, (%rsi) ## encoding: [0x62,0xf4,0x7c,0x08,0x66,0x3e]
```

Comment:
Should to optimize back to legacy encoding? The same below.

Reply:
Would raise another pr to support the encoding compress, currently we only support compress to vex.

Reply:
Unlike NDD instructions, whose non-NDD versions tests are scattered across files,  non-promoted instructions are usually for intrinsics only, only several tests were added for them. 

I think we should merge the tests for promoted instructions with existing one, e.g. x64-cet-intrinsics.ll